flake.nix

```diff
@@ -5,11 +5,11 @@
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
     flake-utils.url = "github:numtide/flake-utils";
     pocket-ic-darwin-gz = {
-      url = "https://download.dfinity.systems/ic/69e1408347723dbaa7a6cd2faa9b65c42abbe861/openssl-static-binaries/x86_64-darwin/pocket-ic.gz";
+      url = "https://download.dfinity.systems/ic/6b7b236790c1fd4e2aeaf3444244b270c7378b02/binaries/x86_64-darwin/pocket-ic.gz";
```

Comment:
Let's use the new links here in the final version, I think `https://github.com/dfinity/pocketic/releases/download/3.0.0/pocket-ic-x86_64-darwin.gz` and `https://github.com/dfinity/pocketic/releases/download/3.0.0/pocket-ic-x86_64-linux.gz`

Reply:
(we'd have to release the new server before that)